pagefetcher.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python2
+
+import urllib
+import os
+from __builtin__ import str
+
+# set of dictionaries to store url's and names
+CISCO = {"name1": "url1", "name2": "url2"}
+TEMPLATE_TO_URL = {"cisco": CISCO}
+
+
+class UrlNotAvailable(Exception):
+    """ Exception class to raise in case of a bad URL. """
+
```

Comment:
I always do and since I'm using eclipse it's automatic.
![snapshot1](https://cloud.githubusercontent.com/assets/15163267/11566535/e4ccbb84-99b0-11e5-9737-9ac472a38289.png)